crates/containerd-shim-wasmedge/src/instance/instance_linux.rs

```diff
@@ -39,11 +39,7 @@ impl LibcontainerInstance for Wasi {
             )
             .unwrap(),
             exit_code: Arc::new((Mutex::new(None), Condvar::new())),
-            stdio: Stdio {
-                stdin: cfg.get_stdin().try_into().unwrap(),
-                stdout: cfg.get_stdout().try_into().unwrap(),
-                stderr: cfg.get_stderr().try_into().unwrap(),
-            },
+            stdio: cfg.try_into().expect("failed to open stdio"),
```

Comment:
This is mostly just readability but this looks a bit strange to me since config is a lot more than just stdio.

Reply:
This does not consume the `cfg` since it's taken by reference.
Do you prefer as it was before?
Maybe something like `Stdio::init_from_config(cfg)` would be better?

Reply:
I think `Stdio::init_from_config(cfg)` would look better